istio/client.go

```diff
@@ -21,7 +21,7 @@ import (
 	"github.com/sirupsen/logrus"
 	"k8s.io/client-go/dynamic"
 	"k8s.io/client-go/kubernetes"
-	_ "k8s.io/client-go/plugin/pkg/client/auth"
+	"k8s.io/client-go/plugin/pkg/client/auth"
```

Comment:
I think we did need the "_" here. Are you sure we want that removed?

Reply:
@girishranganathan  you are right we need "_" but we not using this then why we importing this libary.